### PR TITLE
[NO-JIRA] Help content for injection into UI

### PR DESCRIPTION
**Pull request by cheinz-sag**

### CODE REVIEW

File: content/datahub/integrating-datahub-with-other-products-bundle/integration-powerbi.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+helpcontent:
+  - label: configuring-powerbi-access
+    title: Configuring access to Microsoft Power BI reports
+    content: "**Microsoft Power BI** is a business intelligence tool which allows you to create and use interactive reports for data from various sources. In particular, the reports can consume your IoT data stored in a data lake using Dremio as query and data access layer.
+    
+
+    To view those reports in Cumulocity IoT DataHub, you need an **Azure Active Directory tenant** with an application that can interact with Microsoft Power BI. You neither must sign in to Microsoft Power BI nor need a Microsoft Power BI license to access the reports. For access authentication an **Azure Active Directory service principal** object with an application secret is used.
+    
+
+    Once you have configured the Microsoft Power BI connection, you can access corresponding workspaces and published reports under *Microsoft Power BI* in the *Navigator*."
```
Comment (MWindel) [inline content/datahub/integrating-datahub-with-other-products-bundle/integration-powerbi.md:13]:
```suggestion
    Once you have configured the Microsoft Power BI connection, you can access corresponding workspaces and published reports under **Microsoft Power BI** in the **Navigator**."
```

File: content/datahub/integrating-datahub-with-other-products-bundle/integration-powerbi.md [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+  - label: working-with-reports
+    title: Working with reports
+    content: "Once the Microsoft Power BI settings have been defined, you can work with published reports.
+    
+    
+    Click **Add report** to list all available workspaces and reports. All workspaces which grant member or admin access to the service principal are listed. Select a workspace and then a report from that workspace. You can open multiple reports. A report is closed by clicking **Remove report**.
```
Comment (MWindel) [inline content/datahub/integrating-datahub-with-other-products-bundle/integration-powerbi.md:19]:
```suggestion
    Click **Add report** to list all available workspaces and reports. All workspaces which grant member or admin access to the service principal are listed. Select a workspace and then a report from that workspace. You can open multiple reports. To close a report click **Remove report**. 
```

File: content/datahub/operating-datahub-bundle/checking-system-information.md [hunk 3]
```diff
@@ -2,6 +2,16 @@
 weight: 10
 title: Checking system information
 layout: redirect
+helpcontent:
+  - label: checking-system-information
+    title: Checking system information
+    content: "Under **Microservice** you will find the status of the microservice, indicating via green or red whether the web application can access the microservice.
```
Comment (MWindel) [inline content/datahub/operating-datahub-bundle/checking-system-information.md:8]: How is the color indication done? Icon, whole phrase is colored.... ? 

This should be more specific.

Comment (MWindel) [reply content/datahub/operating-datahub-bundle/checking-system-information.md:8]:
```suggestion
    content: "Under **Microservice** you find the status of the microservice, indicating via green or red whether the web application can access the microservice.
```

File: content/datahub/operating-datahub-bundle/tracking-usage-statistics.md [hunk 4]
```diff
@@ -2,6 +2,14 @@
...
 title: Tracking usage statistics
 layout: redirect
+helpcontent:
+  - label: tracking-usage-statistics
+    title: Tracking usage statistics
+    content: "If enabled, the system tracks **usage statistics** on the amount of data being processed. For offloading queries the amount of data being read from the base collection is tracked. For ad-hoc queries the amount of data being read from the data lake is tracked.
+    
+    The top panels show overall **summary statistics** as well as statistics separated for offloading and ad-hoc queries. The table shows the details on a per-day basis for the selected month. The date of each entry links to the **Query log**, which lists all queries for the respective day.
+    
+    The statistics are refreshed once per hour. They are deleted after a retention period, so for older months statistics may no longer be available."
```
Comment (doernemt) [inline content/datahub/operating-datahub-bundle/tracking-usage-statistics.md:12]: The statistics are refreshed once per hour. They are deleted after a retention period, so for older months statistics may no longer be available."
=> The statistics are UPDATED once per hour. They are deleted after a retention period, so for LONGER PAST months statistics may no longer be available."

File: content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md [hunk 5]
```diff
@@ -2,6 +2,14 @@
 weight: 60
 title: Monitoring offloading jobs
 layout: redirect
+helpcontent:
+  - label: monitoring-offloading-jobs
+    title: Monitoring offloading jobs
+    content: "Once you have configured and started your offloading pipelines, they regularly offload data to the data lake. The execution status of each pipeline run is monitored and provides details on execution schedule, number of offloaded records, runtime and so on. In case an offloading has failed, for example, due to data lake connectivity issues, failure details are provided as well.
```
Comment (doernemt) [inline content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md:8]: Should be briefly mention the alarms feature here?

File: content/datahub/working-with-datahub-bundle/monitoring-compaction-jobs.md [hunk 6]
```diff
@@ -2,6 +2,14 @@
 weight: 70
 title: Monitoring compaction jobs
 layout: redirect
+helpcontent:
+  - label: monitoring-compaction-jobs
+    title: Monitoring compaction jobs
+    content: "During offloading, data from a Cumulocity IoT base collection is written into files in the data lake. In order to ensure a compact physical layout of those files, the system automatically runs periodic **compaction jobs** in the background for each offloading pipeline. The execution status of each compaction run is monitored and provides details on execution schedule, runtime etc.
```
Comment (MWindel) [inline content/datahub/working-with-datahub-bundle/monitoring-compaction-jobs.md:8]:
```suggestion
    content: "During offloading, data from a Cumulocity IoT base collection is written into files in the data lake. In order to ensure a compact physical layout of those files, the system automatically runs periodic **compaction jobs** in the background for each offloading pipeline. The execution status of each compaction run is monitored and provides details on execution schedule, runtime and so on.
```

**Review by MWindel — APPROVED** [review]

**Pull request merged**